test/unit/store/model-store.js

```diff
@@ -84,4 +85,31 @@ describe('ModelStore', () => {
         });
     });
 
+    it('should check out models', () => {
+        // I would pull this into a beforeEach, but it's only used here so far and would mess up the instantiate empty test
```

Comment:
Should instantiate tests are irrelevant and were being removed if causing a problem in the beforeEach. If there is common code in all of the tests which are setting up a model then move it to the beforeEach and kill the instantiate test